api/src/main/scala/WebServer.scala

```diff
@@ -0,0 +1,125 @@
+package org.wikiwatershed.mmw.geoprocessing
+
+import akka.http.scaladsl.unmarshalling.Unmarshaller._
+import spray.json._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import DefaultJsonProtocol._
```

Comment:
Sounds good. I left it in. I adjusted the order of the imports here to keep akka things etc together.